Applications of C++ (#4600)
girlscript/winter-of-contributing: C_CPP/Fundamentals/Applications of CPP.md

# **APPLICATIONS OF C++**

> C++ needs no inroduction to anyone in the modern world. Programmers across the globe have learned the language and have improvised quite innovatively to provide technology for the current and future needs.
> Young programmers are quite keen to learn CPP as their first language and therefore it would be apt to say that CPP is the gateway to the World of Programming! But since it is a back-end language and doesn't provide much output, it becomes easy for them to get bored in the middle of their learning process. To curb this problem it is improtant to know what you can do with CPP and where it is being used in our daily life.

Reviewer: ```suggestion
C++ needs no introduction to anyone in the modern world. Programmers across the globe have learned the language and have improvised quite innovatively to provide technology for the current and future needs.
Young programmers are quite keen to learn CPP as their first language and therefore it would be apt to say that CPP is the gateway to the World of Programming! But since it is a back-end language and doesn't provide much output, it becomes easy for them to get bored in the middle of their learning process. To curb this problem it is important to know what you can do with CPP and where it is being used in our daily life.
```

> \

Reviewer 2: @RaghavvGupta Remove the extra backslashes throughout the document.

> 
> 
> 

Reviewer: ```suggestion
```
Remove unwanted spaces !!

# **ADOBE**
Majority of the applications which are developed by Adobe Systems are developed in CPP programming language like Photoshop, ImageReady, Illustrator and Adobe Premier.

# **GAMING**
CPP is one of the most widely-used programming languages in game development. It has been used to develop games, such as StarCraft and Counter-Strike, and gaming consoles, including Xbox 360, PlayStation, and Nintendo Switch.
Gaming involves lots of resource-intensive functions, whether it’s generating graphics or facilitating multiplayer sessions. CPP helps in optimizing the use of resources, such as allowing programmers to adjust exactly how data structures and memory allocation are handled.

# **IoT Devices**
IoT (Internet of Things) devices include embedded systems which mostly rely on CPP. Some examples of these devices are appliances, IoT enabled cars, smartwatches, smart TVs, and medical devices. 

> Here, I have listed a few real world applications of CPP primarily focussing on young developers. Apart from these CPP is also used in Movie production, Graphic designing, Medical technology, aeroplane softwares, AR/VR, machine lerning etc and therefore it is adequate to say that CPP is and will be one of the most important languages to learn in the coming years.

Reviewer: ```suggestion
Here, I have listed a few real world applications of CPP primarily focussing on young developers. Apart from these CPP is also used in Movie production, Graphic designing, Medical technology, aeroplane softwares, AR/VR, machine learning etc, and therefore it is adequate to say that CPP is and will be one of the most important languages to learn in the coming years.
```
